react-routing/client/src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import { Header } from './components/Header';
+import { Catalog } from './pages/Catalog';
+import { About } from './pages/About';
+import { ProductDetails } from './pages/ProductDetails';
+
+export function App() {
+  return (
+    <div>
```

Comment:
This outer div isn't needed and probably doesn't have an impact on the layout. It's usually left off.